app/transformations/LandOrPropertyTransformer.scala

```diff
@@ -211,7 +211,7 @@ class LandOrPropertyTransformer @Inject() extends Transformer {
               prop => {
                 OtherRecipientDetailsPage(srn, index, LandOrPropertySeller) -> RecipientDetails(
                   heldPropertyTransaction.optPropertyAcquiredFromName.get,
-                  prop.otherDescription.get
+                  prop.otherDescription.getOrElse(None)
```

Comment:
would it be better if we use :
prop.otherDescription instead prop.otherDescription.getOrElse(None)

Reply:
now it's changed :)